SecurityReviewBuddy.py

```diff
@@ -0,0 +1,37 @@
+from langchain.output_parsers import ResponseSchema
+from langchain_community.llms.ollama import Ollama
+from langchain_core.prompts import ChatPromptTemplate
+from langchain_core.output_parsers import StrOutputParser
+from langchain_openai import ChatOpenAI
+
+
+class SecurityReviewBuddy(object):
```

Comment:
Probably needs code refactoring so we make this generic and just change the prompt, but I am not a Python developer so I did the fasted way